src/components/routes/Book.js

```diff
@@ -0,0 +1,15 @@
+import Input from './Input';
+
+function Book() {
+  return (
+    <>
+      <div>
+        <h2>Awaken the giant within</h2>
+        <p>Anthony Robbins</p>
+      </div>
+      <Input />
+    </>
+  );
```

Comment:
- [ ] `Input` component is a form, kindly render it from to `Books` component. Otherwise for each book that you have Book component will be printing 1 form: 

```
import Book from './Book';
import Input from './Input';

const Books = () => (
  <>
    <h1>Book List</h1>
    <Book />
    <Input />
  </>
);

export default Books;
```

- [ ] OPTIONAL: I would also recommend naming Input component more descriptively, eg: `CreateNewBook`. As your application scales, you may be confused thinking which exactly input is it 😉